### PR TITLE
Small fixes in Dutch language file

### PR DESCRIPTION
**Pull request by pe7er**
I tried to improve the Dutch language file.
E.g. I changed the translation of "participant" from "lid" to "deelnemer".

### CODE REVIEW

File: lang/main-nl.json [hunk 1]
```diff
@@ -232,7 +232,7 @@
         "screenSharingFailedToInstallTitle": "Installatie van extensie voor het delen van het scherm is mislukt",
         "screenSharingFirefoxPermissionDeniedError": "Er is iets misgegaan tijdens het delen van uw scherm. Controleer of u hier toestemming voor hebt verleend.",
         "screenSharingFirefoxPermissionDeniedTitle": "Scherm delen kan niet worden gestart.",
-        "screenSharingPermissionDeniedError": "Er is iets misgegaan met de machtigingen voor de extensie voor het delen van het scherm. Laad de toepassing opnieuw en probeer het nog eens.",
+        "screenSharingPermissionDeniedError": "Oeps! Er is iets misgegaan met de uitbreidingsrechten voor het delen van het scherm. Laad de toepassing opnieuw en probeer het nog eens.",
```
Comment (rubenk) [inline lang/main-nl.json:235]: I'd leave the "Oeps!" out.
"permissies" is wellicht  duidelijker dan "uitbreidingsrechten"

File: lang/main-nl.json [hunk 2]
```diff
@@ -248,7 +248,7 @@
         "stopRecording": "Opname stoppen",
         "stopRecordingWarning": "Weet u zeker dat u de opname wilt stoppen?",
         "stopStreamingWarning": "Weet u zeker dat u de livestream wilt stoppen?",
-        "streamKey": "Sleutel voor livestream",
+        "streamKey": "Livestream sleutel",
```
Comment (rubenk) [inline lang/main-nl.json:251]: Deze samentrekking hoort of aan elkaar, of met een koppelstreepje ertussen, maar persoonlijk vind ik "Sleutel voor livestream" duidelijker.

File: lang/main-nl.json [hunk 3]
```diff
@@ -271,7 +271,7 @@
         "bad": "Slecht",
         "detailsLabel": "We horen er graag meer over.",
         "good": "Goed",
-        "rateExperience": "Beoordeel uw ervaring tijdens de vergadering",
+        "rateExperience": "Beoordeel uw vergaderervaring",
```
Comment (rubenk) [inline lang/main-nl.json:274]: The original translation seems clear to me.

File: lang/main-nl.json [hunk 4]
```diff
@@ -337,13 +337,13 @@
...
         "toggleChat": "Chatgesprek openen of sluiten",
-        "toggleFilmstrip": "Videominiaturen weergeven of verbergen",
+        "toggleFilmstrip": "Toon of verberg videominiaturen",
         "toggleScreensharing": "Schakelen tussen camera en het delen van het scherm",
         "toggleShortcuts": "Sneltoetsen weergeven of verbergen",
         "videoMute": "Uw camera starten of stoppen"
     },
     "liveStreaming": {
-        "busy": "Er worden streamingresources vrijgemaakt. Probeer het over enkele minuten opnieuw.",
+        "busy": "We werken aan het vrijmaken van streaming middelen. Probeer het over enkele minuten opnieuw.",
```
Comment (rubenk) [inline lang/main-nl.json:346]: Either streamingmiddelen or streaming-middelen

File: lang/main-nl.json [hunk 5]
```diff
@@ -512,7 +512,7 @@
         "alertOk": "OK",
         "alertTitle": "Waarschuwing",
         "alertURLText": "De ingevoerde server-URL is ongeldig",
-        "buildInfoSection": "Buildgegevens",
+        "buildInfoSection": "Versie gegevens",
```
Comment (rubenk) [inline lang/main-nl.json:515]: Versiegegevens

File: lang/main-nl.json (hunk 2 above)
Comment (pe7er) [reply lang/main-nl.json:251]: Fair enough. I'll change it back...

File: lang/main-nl.json (hunk 4 above)
Comment (pe7er) [reply lang/main-nl.json:346]: tnx!

File: lang/main-nl.json (hunk 5 above)
Comment (pe7er) [reply lang/main-nl.json:515]: tnx!

**Comment by saghul** [comment]
Dank je wel folks! Happy to merge once you both agree on the language details. Greetings from 020 :-)

**Comment by pe7er** [comment]
Thanks @rubenk for your feedback!

@saghul I've added a new PR with the suggestions that Ruben gave me. Greetings from 024 :-)

**Comment by saghul** [comment]
@rubenk Can you give it an approval?

**Comment by rubenk** [comment]
LGTM! Greetings from 070 😁

**Comment by saghul** [comment]
Thanks folks!

**Comment by rubenk** [comment]
Thank you!

**Comment by pe7er** [comment]
Thanks!

**Pull request merged**